app/proguard-rules.pro

```diff
@@ -73,6 +73,9 @@
 -keep @interface pm.gnosis.heimdall.data.adapters.DecimalNumber
 -keep @interface pm.gnosis.heimdall.data.adapters.BigDecimalNumber
 
+-keep interface pm.gnosis.ethereum.rpc.models.** { *; }
+-keep class pm.gnosis.ethereum.rpc.models.** { *; }
```

Comment:
Just out of curiosity: Why do we need to keep this now and not before? What changed?

Reply:
We have updated moshi and kotlin versions. My assumption is that either class generation syntax changed, which lead to this obfuscation error or that ens resolution was not working before in the first place.